src/sagemaker_algorithm_toolkit/hyperparameter_validation.py

```diff
@@ -243,16 +244,34 @@ def _visit(name, visited, stack):
 
         return dependencies_stack
 
+    def _declare_alias(self, key_name, alias_name):
+        if key_name not in self.hyperparameters:
+            raise exc.AlgorithmError("Key name { " + key_name + " } does not exist in list of hyperparameters")
```

Comment:
please use the formatter to format the string. See other exceptions in the codebase.